pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -1,20 +1,61 @@
 import logging
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Tuple, Union
 
-from typing import Union, List
 from enforce_typing import enforce_types
 
 from pdr_backend.lake.duckdb_data_store import DuckDBDataStore
 from pdr_backend.lake.payout import Payout
 from pdr_backend.lake.prediction import Prediction
 from pdr_backend.lake.subscription import Subscription
+from pdr_backend.pdr_dashboard.util.data import (
+    col_to_human,
+    filter_objects_by_field,
+    get_feed_column_ids,
+    get_feeds_stat_with_contract,
+    get_feeds_subscription_stat_with_contract,
+)
+from pdr_backend.pdr_dashboard.util.format import format_dict, format_table
+from pdr_backend.pdr_dashboard.util.prices import (
+    calculate_tx_gas_fee_cost_in_OCEAN,
+    fetch_token_prices,
+)
 from pdr_backend.util.constants_opf_addrs import get_opf_addresses
 
 logger = logging.getLogger("predictoor_dashboard_utils")
 
 
+# pylint: disable=too-many-instance-attributes
 class DBGetter:
-    def __init__(self, lake_dir: str):
-        self.lake_dir = lake_dir
+    def __init__(self, ppss):
+        self.lake_dir = ppss.lake_ss.lake_dir
+        self.network_name = ppss.web3_pp.network
+
+        # fetch token prices
+        self.fee_cost = calculate_tx_gas_fee_cost_in_OCEAN(
```

Comment:
I don't think this should be inside dbGetter, it has nothing to do with db data. Let's movi it back to app

Reply:
What I mean is, in general if I want to access the fee_costs I wouldn't expect it do be in the db

Reply:
> What I mean is, in general if I want to access the fee_costs I wouldn't expect it do be in the db

I want to rename db_getter either way, to data manager. I think that is more reflective of what it's doing right now. I think it fits then :)